console/src/routes.tsx

```diff
@@ -148,8 +150,8 @@ const routes = store => {
           {uiKitRouter}
         </Route>
       </Route>
-      <Route path="404" component={PageNotFound} status="404" />
-      <Route path="*" component={PageNotFound} status="404" />
+      <Route path="404" component={PageNotFound} />
+      <Route path="*" component={PageNotFound} />
```

Comment:
Remove `status`, because typescript complains that `status` is not defined on the component